mr/src/test/java/org/apache/iceberg/mr/hive/serde/objectinspector/TestIcebergObjectInspector.java

```diff
@@ -168,26 +161,16 @@ public void testIcebergObjectInspector() {
     assertThat(timestampField.getFieldID()).isEqualTo(11);
     assertThat(timestampField.getFieldName()).isEqualTo("timestamp_field");
     assertThat(timestampField.getFieldComment()).isEqualTo("timestamp comment");
-    if (HiveVersion.min(HiveVersion.HIVE_3)) {
-      assertThat(timestampField.getFieldObjectInspector().getClass().getSimpleName())
-          .isEqualTo("IcebergTimestampObjectInspectorHive3");
-    } else {
-      assertThat(timestampField.getFieldObjectInspector())
-          .isEqualTo(IcebergTimestampObjectInspector.get());
-    }
+    assertThat(timestampField.getFieldObjectInspector().getClass().getSimpleName())
+        .isEqualTo("IcebergTimestampObjectInspector");
 
     // timestamp with tz
     StructField timestampTzField = soi.getStructFieldRef("timestamptz_field");
     assertThat(timestampTzField.getFieldID()).isEqualTo(12);
     assertThat(timestampTzField.getFieldName()).isEqualTo("timestamptz_field");
     assertThat(timestampTzField.getFieldComment()).isEqualTo("timestamptz comment");
-    if (HiveVersion.min(HiveVersion.HIVE_3)) {
-      assertThat(timestampTzField.getFieldObjectInspector().getClass().getSimpleName())
-          .isEqualTo("IcebergTimestampWithZoneObjectInspectorHive3");
-    } else {
-      assertThat(timestampTzField.getFieldObjectInspector())
-          .isEqualTo(IcebergTimestampWithZoneObjectInspector.get());
-    }
+    assertThat(timestampTzField.getFieldObjectInspector().getClass().getSimpleName())
```

Comment:
we can probably just use the class directly (similarly to what has been previously done for Hive2:

```
assertThat(timestampTzField.getFieldObjectInspector())
          .isEqualTo(IcebergTimestampWithZoneObjectInspector.get());
```